packages/allow-scripts/src/config.js

```diff
@@ -13,7 +13,14 @@ const bannedBins = new Set(['corepack', 'node', 'npm', 'pnpm', 'yarn'])
 /**
  * Scripts with names other than these are ignored and unenforced by default.
  */
-const DEFAULT_LIFECYCLE_EVENTS = ['preinstall', 'install', 'postinstall']
+const DEFAULT_LIFECYCLE_EVENTS = [
+  'install',
+  'postinstall', 'postinstallonly', 'preinstall', 'preinstallonly',
+  'preaudit', 'preauditonly', 'postaudit', 'postauditonly',
+  'prepack', 'prepackonly', 'postpack', 'postpackonly',
+  'preprepare', 'preprepareonly', 'postprepare', 'postprepareonly',
+  'prepublish', 'prepublishonly', 'postpublish', 'postpublishonly',
+];
```

Comment:
Does this account for case-sensitivity? Do _package managers_ care about case-sensitivity?  I've only ever seen `prepublishOnly`, for instance.

Reply:
Hm, I thought I addressed that part but looks like I forgot it. Good catch!